app/controllers/admin/vote_users_controller.rb

```diff
@@ -11,7 +11,7 @@ def show
     @user = User.find(params[:id])
     @votes = Vote.with_deleted
```

Comment:
Aa.

In the `render 'log'`, perhaps it would be more clear to pass the instance-variables in as parameters here:
`render 'log', votes: @votes` and it's more clear 😄